Release_Notes.md

```diff
@@ -14,11 +14,12 @@ See the License for the specific language governing permissions and<BR>
 limitations under the License.*_<BR>
 
 # **Release Notes** #
-_*Last Update: 2023/09/13*_
```

Comment:
I think though you shouldn't bring the RTL to 0.9

You should bring the RTL to 1.0-rc1

Reply:
Okay, I can make this change. Perhaps I missed the intent behind the 0.9 versioning we discussed last Friday. Was that just for Firmware?

Reply:
For ROM. RTL we believe to be 1.0-rc1